ci-operator/config/stackrox/kernel-packer/stackrox-kernel-packer-ddolgov-feature-repackage-cache.yaml

```diff
@@ -0,0 +1,93 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: kernel-crawling-periodic
+  cron: 15 */8 * * *
+  steps:
+    post:
+    - as: delete-vm
+      commands: .openshift-ci/crawler/post.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-collector-driver-push
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
+    pre:
+    - as: create-vm
+      commands: .openshift-ci/crawler/pre.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-collector-driver-push
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
+    test:
+    - as: crawling
+      commands: .openshift-ci/gcp/command.sh .openshift-ci/crawler/crawl.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-collector-driver-push
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
+      timeout: 6h0m0s
+    - as: sync
+      commands: .openshift-ci/gcp/command.sh .openshift-ci/crawler/sync.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-collector-driver-push
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
+      timeout: 6h0m0s
+    - as: repackaging
+      commands: .openshift-ci/gcp/command.sh .openshift-ci/crawler/repackage.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-collector-driver-push
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
+      timeout: 6h0m0s
+    - as: cache
+      commands: .openshift-ci/gcp/command.sh .openshift-ci/crawler/cache.sh
+      credentials:
+      - mount_path: /tmp/secret/stackrox-kernel-packer-crawl
+        name: stackrox-collector-driver-push
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 4000m
+          memory: 4000Mi
+      timeout: 6h0m0s
```

Comment:
What is the current behaviour when the timeout is hit? do we lose the work done by the job and the job fails? Just wondering if we could use `grace_period` to save the work we have done so far

Reply:
Yep, if we hit the limit, the vm will be deleted with all the intermediate results. This suggestion would be good for some of the follow-up tasks.